pkg/controller.v1alpha3/consts/const.go

```diff
@@ -126,4 +135,11 @@ var (
 	DefaultKatibNamespace = env.GetEnvOrDefault(DefaultKatibNamespaceEnvName, "kubeflow")
 	// DefaultComposer is the default composer of katib suggestion.
 	DefaultComposer = env.GetEnvOrDefault(DefaultKatibComposerEnvName, "General")
+
+	// DefaultKatibDBManagerServiceNamespace is the default namespace of Katib DB Manager
+	DefaultKatibDBManagerServiceNamespace = env.GetEnvOrDefault(DefaultKatibDBManagerServiceNamespaceEnvName, DefaultKatibNamespace)
```

Comment:
LGTM but I think we should illustrate the relationship between DefaultKatibNamespace and DefaultKatibDBManagerServiceNamespace in the doc in the future.

Reply:
Sure, I will create doc about all env variables in Katib components today